vmm/sandbox/src/sandbox.rs

```diff
@@ -152,6 +153,9 @@ pub struct KuasarSandbox<V: VM> {
     pub(crate) client: Arc<Mutex<Option<SandboxServiceClient>>>,
     #[serde(skip, default)]
     pub(crate) exit_signal: Arc<ExitSignal>,
+    #[serde(skip, default)]
+    pub(crate) sandbox_cgroups: SandboxCgroup,
+    pub(crate) sandbox_cgroup_path: String,
```

Comment:
Should we keep this field? Perhaps remove it?

Reply:
SandboxCgroup is an attribute of a sandbox object, which is necessary to be stored in the sandbox object.

Reply:
It never be read. Only usage for write access.

Reply:
`sandbox_cgroup_path` is used in the `KuasarSandbox::recover()` function which is added in this PR now.
After vmm-sandboxer process is restarted, we need to make sure `sandbox_cgroups` object can be restricted correctly with `sandbox_cgroup_path`, otherwise sandbox cgroup dir will be left when the sandbox is deleted.

Reply:
So how about move this String into `sandbox_cgroups` object and skip all child object excepts this String.

Reply:
Nice suggestion! I have updated the code.